test/mobile/features/scripts/build_fixture.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+if [ -z "$UNITY_VERSION" ]
+then
+  echo "UNITY_VERSION must be set"
+  exit 1
+else
+  export PATH="/Applications/Unity/Hub/Editor/$UNITY_VERSION/Unity.app/Contents/MacOS:$PATH"
+  echo "\`which Unity\`=`which Unity`"
+fi
+
+pushd "${0%/*}"
+  script_path=`pwd`
+popd
+
+pushd "$script_path/../fixtures"
+
+# Run unity and immediately exit afterwards, log all output, disable the
+# package manager (we just don't need it and it slows things down)
+DEFAULT_CLI_ARGS="-quit -batchmode -logFile unity.log -noUpm"
+project_path=`pwd`/maze_runner
+
+# Creating a new project in the MyProject directory
+Unity $DEFAULT_CLI_ARGS -createProject $project_path
+
+# Installing the Bugsnag package
+echo "Importing Bugsnag.unitypackage into $project_path"
+Unity $DEFAULT_CLI_ARGS -projectPath $project_path -importPackage $script_path/../../../../Bugsnag.unitypackage
```

Comment:
`Unity` isn't in my path - is there any way this could work like the way Rakefile calculates the location of Unity Hub?

Reply:
I'll follow this PR up straight away to make the build Rake driven, so it should make better use of the existing intelligence.